src/software/geom/voronoi_util.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <boost/polygon/voronoi.hpp>
+
+#include "software/geom/circle.h"
+#include "software/geom/point.h"
+#include "software/geom/rectangle.h"
+
+using boost::polygon::voronoi_builder;
+using boost::polygon::voronoi_diagram;
+using boost::polygon::voronoi_vertex;
+
+struct VoronoiDiagram
+{
+    explicit VoronoiDiagram(std::vector<Point> points)
```

Comment:
we probably want to delete the default constructor as well, like:
```
VoronoiDiagram() = delete;
```

Reply:
done